src/app/common/o/services/oLms.service.ts

```diff
@@ -0,0 +1,66 @@
+import { Injectable } from "@angular/core";
+import { HttpClient } from "@angular/common/http";
+import { environment } from "src/environments/environment";
+
+@Injectable({
+  providedIn: "root",
+})
+export class OLmsService {
+  constructor(private httpClient: HttpClient) {}
+
+  get(url: string) {
+    return this.httpClient.get(url);
+  }
+
+  put(url: string, body: any = "", options: object = {}) {
+    return this.httpClient.put(url, body, options);
+  }
+
+  setBorrowerNameOf(input: any) {
+    this.httpClient
+      .get(
+        `${environment.adminBackendUrl}${environment.readBorrowerUri}/${input.cardNo}`
+      )
+      .subscribe(
+        (result: any[]) => (input.borrowerName = result[0].name),
+        (error: any) => {
+          input.borrowerName = "(error retrieving borrower name)";
+          // do something with logger here
+        }
+      );
+  }
+
+  setBranchNameOf(input: any) {
```

Comment:
I heard you mention that you would like move this to another file so that it can be re-used. I think all of us could make use of these function if you want to collaborate on where to put them in the service folder. I know I will need this same functionality for the Borrowers.

Reply:
I'm not planning to put stuff into another file, but rather add it to a more general branch that'll be an ancestor of my feature branches.

That's a good idea, but my only concern is that I'm assuming we're still each expected to build our own features from top to bottom. I don't know if that means we can't use the same service modules, but I made my own subdirectory in src/app/common to be on the safe side.